garb/garb_raii.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) 2020 Percona LLC and/or its affiliates. All rights reserved.
+
+   This program is free software; you can redistribute it and/or
+   modify it under the terms of the GNU General Public License
+   as published by the Free Software Foundation; version 2 of
+   the License.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software
+   Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+#ifndef GARB_RAII_INCLUDED
+#define GARB_RAII_INCLUDED 1
+#include "garb_gcs.hpp"
+
+/*
+  RAII class for freeing of gcs action objects
```

Comment:
It frees act buffer, not action object.
gcs_action_buffer_guard?

Reply:
Right! Will change it to `Gcs_action_buffer_guard` in next revision.

Reply:
Done